app/models/result.rb

```diff
@@ -0,0 +1,9 @@
+Result = Struct.new(:success) {
```

Comment:
This isn't modelling any data so there is an argument that it shouldn't be in models. Perhaps it should be in services/ or lib/ ?